routes/impersonation.ts

```
@@ -0,0 +1,9 @@
+import express from "express";
+const router = express.Router();
+import authenticate from "../middlewares/authenticate";
+import { updateImpersonationRequestValidator } from "../middlewares/validators/impersonationRequests";
+import { updateImpersonationRequestStatusController } from "../controllers/impersonationRequests";
+
+router.patch("/requests/:id", authenticate,updateImpersonationRequestValidator,updateImpersonationRequestStatusController);
```

ESLint

[error] Replace `"/requests/:id",·authenticate,updateImpersonationRequestValidator,updateImpersonationRequestStatusController` with `⏎··"/requests/:id",⏎··authenticate,⏎··updateImpersonationRequestValidator,⏎··updateImpersonationRequestStatusController⏎`

(prettier/prettier)